src/js/Content/Features/Community/BoosterCreator/FShowGemsCost.js

```diff
@@ -4,16 +4,22 @@ import {Page} from "../../Page";
 export default class FShowGemsCost extends Feature {
 
     apply() {
+
+        // Localized string of "(quantifier) + Gems"
+        const gemWord = document.querySelector(".booster_creator_goostatus .goo_display").textContent.trim().replace(/[\d]+,?/g, "");
+
         Page.runInPageContext(gemWord => {
-            /* eslint-disable no-invalid-this */
-            const jq = window.SteamFacade.jq;
-            jq("#booster_game_selector option").each(function() {
-                const val = jq(this).val();
+
+            // CBoosterCreatorPage cannot be transferred via postMessage()
```

Comment:
It can be transferred, we were just using the wrong identifier (`CBoosterCreator` instead of `CBoosterCreatorPage`

Reply:
I tried to, but it throws `Uncaught DOMException: Failed to execute 'postMessage' on 'Window': HTMLDivElement object could not be cloned.` Anyway, I found a workaround.

Reply:
Well, the whole `CBoosterCreatorPage` can't be cloned but we only need `sm_rgBoosterData` which is absolutely cloneable

Reply:
I couldn't get it to work without the workaround, `window.postMessage(CBoosterCreatorPage.sm_rgBoosterDatam, "*")` throws as well...or am I missing something?

Reply:
It seems like I was talking about using `CBoosterCreatorPage.sm_rgBoosterData` in the page context, which would work because it doesn't need to be cloned.
You were talking about transferring it to the extension context